mysql-test/suite/rpl/t/rpl_percona_bug995624.test

```diff
@@ -0,0 +1,46 @@
+source include/master-slave.inc;
+source include/not_gtid_enabled.inc;
+
+connection slave;
+set global userstat=ON;
+--echo Rows_updated on slave - starting situation
+select ROWS_UPDATED from information_schema.client_statistics;
+
+connection master;
+set global userstat=ON;
+--echo Rows_updated on master - starting situation
+select ROWS_UPDATED from information_schema.client_statistics;
+
+create table t1 (m int);
+--echo 3 "updates" to rows
+insert into t1 values(15),(16),(17); 
+--echo 1 "update" to rows (4 in total)
+update t1 set m=20 where m=16;
+--echo 1 "update" to rows (5 in total)
+delete from t1 where m=17;
+create table t2 (n int);
```

Comment:
t2 test looks redundant - please test instead, that a table, that is not replicated by the slave-side replication filtering rules, does not cause userstat bump